java/README.md

```diff
@@ -0,0 +1,248 @@
+# JAVA SDK
+
+Java SDK is a subproject under GraphScope, presenting an efficient Java SDK for GraphScope analytical engine.
+Powered By [alibaba-FastFFI](https://github.com/alibaba/fastFFI), which is able to bridge the huge programming Gap Between Java and C++, Java PIE
```

Comment:
The capitalized `Gap` looks strange, if you want to emphasize, use `**gap**` instead, as we are writting markdown.

Reply:
Done.